flags.go

```diff
@@ -336,15 +335,6 @@ func newApp() (app *cli.App) {
 				Name:  "debug_mutex",
 				Usage: "Print debug messages when a mutex is held too long.",
 			},
-
-			/////////////////////////
-			// Client
-			/////////////////////////
-
-			cli.BoolTFlag{
-				Name:  "enable-storage-client-library",
-				Usage: "If true, will use go storage client library otherwise jacobsa/gcloud",
```

Comment:
please remove enable_storage_client_library from tools/mount_gcsfuse/main.go to remove support from mount command.

Reply:
Also, I would suggest to do git grep for "enable_storage_client_library", "enable-storage-client-library" and "EnableStorageClientLibrary" to see all the usages and remove all of them (even in comments, docs).

Reply:
Thanks fixed